modules/Administration/language/en_us.lang.php

```diff
@@ -839,7 +839,7 @@
     'LBL_AOP_JOOMLA_ACCOUNT_CREATION_EMAIL_TEMPLATE' => 'Joomla Support Portal Account Creation Template',
 
     'LBL_AOP_BUSINESS_HOURS_SETTINGS' => 'Business Hours',
-    'LBL_AOP_BUSINESS_HOURS_DESC' => 'Change the business hours',
+    'LBL_AOP_BUSINESS_HOURS_DESC' => 'Restrict \'Only In The Scheduler\' Workflows to launch only in certain days and times',
```

Comment:
Use single and double quotes when appropriate. You shouldn't need to escape quotes within a string.

Reply:
Ok, I fixed this one and another similar case in the same file. Did I do it right?